### ISSUE TITLE
[IMPROVE] Crafting

### ISSUE THREAD

**Issue by handicraftsman** [DarkholmeTenk/TardisModIssues#233]
Add crafting table and vortex oven to the engine (at the bottom)

**Comment by FoxPotato** [comment]
Or you place them yourself. These seem like additions which arent really necessary.

**Comment by DarkholmeTenk** [comment]
Vortex oven idea I like. Not saying I'm going to do it, but it amuses me.

**Comment by liftoff** [comment]
I don't mind placing my own furnaces and crafting table but it would be nice if we could get craftable versions that match the TARDIS color schemes.  Or perhaps we could just drop furnaces and crafting tables into the lab to have it output nice white versions of each?  That'd be :cool:

**Comment by FoxPotato** [comment]
Yeah. That does sound cool. Blocks kind of stand out against the white console. Even when coloured

**Comment by Soyaro666** [comment]
A lab-like furnace maybe? so it act's like a furnace (maybe consuming artron energy instead of usual fuel) but is used like the lab (withoug GUI)...?

**Comment by FoxPotato** [comment]
Or just use the lab, with a chance when u cook chicken there's a chance a chicken mob spawns ;p
Ofc if this is possible, idk how mob spawning works and shit :p

**Comment by Soyaro666** [comment]
I guess in case of the chicken it could simply be like the lab uses an egg each time raw chicken is cooked. more complicated would be the same with Cows and Pigs since there are only creative spawneggs that could be triggered. that should be as simple as getting a random number and if it's in a specified range the spawnegg is triggered...
But I would expect trouble with this idea since iron becomes steel in some mods which could cause a conflict with chronosteel (lets say I want to make a stack of chronosteel and am getting only a half while the other half became usual steel while I was busy starting the Tardis). that's why I suggested an own machine instead of adding this function to the lab.
